### ISSUE TITLE
Does the "Move" hotkey still work?

### ISSUE THREAD

**Issue by robandstuff** [stockto2/desktophero#119]
The "intro" video says that you can press R to rotate, S to scale, or G(or Z?) to move. Neither G nor Z seem to do anything consistently, although sometimes it seems like triggers either a move or scale-type action.

**Comment by stockto2** [comment]
Press G and then an axis (x, y or z) to move along. That needs to make it
into the documentation eventually.

**Comment by robandstuff** [comment]
Ahhhhh... That makes sense! Do rotate and scale work this way too? If not can they?

**Comment by stockto2** [comment]
Rotate and scale also work along an axis.

**Comment by stockto2** [comment]
I never quite got move working without an axis specified, like you can do
in Blender.

**Comment by robandstuff** [comment]
Sorry, what I meant was can you manually select an axis to rotate/scale along? Rotate seems to auto-select an axis, and scale seems to work in all 3 at the same time.

**Comment by stockto2** [comment]
Right, just hit X, Y or Z after R or S and it should scale along that axis.

**Comment by robandstuff** [comment]
Ah, excellent. Thanks for the help.
